src/utils/net.rs

```diff
@@ -18,6 +22,35 @@ pub fn global_http_client() -> reqwest::Client {
     CLIENT.clone()
 }
 
+/// Download a file via IPFS HTTP gateway in trustless mode.
+/// See <https://github.com/ipfs/specs/blob/main/http-gateways/TRUSTLESS_GATEWAY.md>
+pub async fn download_ipfs_file_trustlessly(
+    cid: &Cid,
+    gateway: Option<&str>,
+    destination: &Path,
+) -> anyhow::Result<()> {
+    // https://docs.ipfs.tech/concepts/ipfs-gateway/
+    const DEFAULT_IPFS_GATEWAY: &str = "https://ipfs.io/ipfs/";
```

Comment:
we could reduce the scope of this as well, given it's not used outside of the URL creation.

Reply:
Fixed